src/contentScript/errors.ts

```diff
@@ -20,7 +20,7 @@
 import { reportError } from "@/telemetry/logging";
 
 function addErrorListeners(): void {
-  window.addEventListener("error", function (error) {
+  window.addEventListener("error", function (error: any) {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7747711)
```suggestion
  window.addEventListener("error", function (error: unknown) {
```